src/Microsoft.Extensions.Configuration.AzureAppConfiguration/Constants/RequestTracingConstants.cs

```diff
@@ -25,6 +25,7 @@ internal class RequestTracingConstants
         public const string DevEnvironmentValue = "Dev";
         public const string KeyVaultConfiguredTag = "UsesKeyVault";
         public const string KeyVaultRefreshConfiguredTag = "RefreshesKeyVault";
+        public const string FeatureManagementV2SchemaEnabledTag = "UsesFeatureManagementV2Schema";
```

Comment:
For the tracing, I would suggest we have something like "FMSchema=v2preview" instead of a boolean. This works better in case we have more versions. It also avoids changing tracing logic when we change the opt-in/opt-out logic.